inginious/frontend/plugins/UNCode/pages/data_policy.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+""" Data policy """
```

Comment:
Can you a more descriptive comment? and set the comment inside the class rather than here.